src/test/java/org/humancellatlas/ingest/audit/AuditTypeTest.java

```diff
@@ -0,0 +1,28 @@
+package org.humancellatlas.ingest.audit;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertEquals;
+
+@JsonTest
+public class AuditTypeTest {
```

Comment:
I am not sure we need this test.
You are only testing the `ObjectMapper` class with this. We don't need to test that class.
Am I wrong?

Reply:
I created this to test to test if de-serialization and serialization work as expected, following the existing test for [WranglingState](https://github.com/ebi-ait/ingest-core/blob/dev/src/test/java/org/humancellatlas/ingest/project/WranglingStateTest.java)

Is this unnecessary? I was following the existing pattern test

Reply:
This actually tests that the text in the enum value matches what's in the text.
The question to be asked is "what is the purpose of this test"?
The fact that some code has been used in one test does not automatically says that code should be replicated.